config/config.go

```diff
@@ -155,6 +155,7 @@ func init() {
 
 	cfg.SetDefault("ovs.ovsdb", "unix:///var/run/openvswitch/db.sock")
 	cfg.SetDefault("ovs.oflow.enable", false)
+	cfg.SetDefault("ovs.oflow.openflow_versions", []string{"OpenFlow10", "OpenFlow13"})
```

Comment:
It seems to fail on Fedora. It works if I set the default to []string{"OpenFlow10"}

Reply:
Does it mean Fedora uses utilities from an ovs 2.2 or earlier (http://docs.openvswitch.org/en/latest/faq/openflow/) ? From my tests, even if you force your bridge to support only OF10, as it is a negotiation, you have the OF10 behaviour but it does not fail.

I will put the default on the safe side with only OpenFlow10 activated. I have not found a way to reliably detect the ovs version and compute openflow_versions from it.